internal/state/db/statepsql/members.go

```diff
@@ -198,6 +199,24 @@ ORDER BY last_updated desc nulls last limit 1
 	return *(*[]byte)(unsafe.Pointer(&usr)), nil
 }
 
+func (db *db) GetUsersDiscordIdAndUsername(ctx context.Context, userIDs []int64) ([]state.UserAndData, error) {
+	q := `
+	SELECT
+		data->'user'->>'id' AS id,data->'user'->>'username' AS user
```

Comment:
`AS username` if change in struct `UserAndData`

Reply:
accepted